next.config.js

```diff
@@ -37,11 +37,16 @@ module.exports = withMDX({
       return [
         {
           source: '/:path+/',
-          permanent: false,
+          permanent: true,
           destination: '/:path+'
         },
         {
-          source: '/api/(.*)',
+          source: '/api',
+          permanent: true,
+          destination: '/docs/api'
+        },
+        {
+          source: '/api(.*)',
```

Comment:
```suggestion
          source: '/api/(.*)',
```
We don't want `/apis` to match